mautic/mautic
# Fixed push to Salesforce via form submit issue that pushed old data instead of new from the form

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | y
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) |
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

When pushing to SF through a form submit, existing data prior to the submit was sent for the contact rather than what was just updated through the form itself. 

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Setup the SF plugin with push to integration enabled and ensure the field mappings are pointing toward the integration field
2. Create a form with last name and email both mapped to the respective contact fields
3. Submit the form
4. Check SF that the person was created as a Lead
5. Submit the form again with a different last name
6. Check SF and notice the last name did not get updated
7. Submit a third time and check SF and note the last name was updated

#### Steps to test this PR:
1. Submit the form with a new email address to create a new Lead in Mautic and SF
2. Submit the form a second time with a different last name
3. Check SF and notice the last name was updated.
